pkg/kubectl/cmd/util/factory.go

```diff
@@ -176,7 +177,25 @@ func NewFactory(optionalClientConfig clientcmd.ClientConfig) *Factory {
 			if cfg.GroupVersion != nil {
 				cmdApiVersion = *cfg.GroupVersion
 			}
-
+			client, err := clients.ClientForVersion(&unversioned.GroupVersion{Version: "v1"})
+			if err != nil {
+				fmt.Printf("Failed to create client: %v", err)
+				return nil, nil
```

Comment:
> Seems like I should return an error here.

Oh, I see, no error available.  `util.HandleError` then or adjust the signature.